build.psm1

```diff
@@ -289,7 +289,14 @@ function Start-ScriptAnalyzerBuild
 function Test-ScriptAnalyzer
 {
     [CmdletBinding()]
-    param ( [Parameter()][switch]$InProcess, [switch]$ShowAll )
+    param (
+        [Parameter()]
+        [switch]$InProcess,
+
+        [switch]$ShowAll,
+
+        [string]$RuleToTest
```

Comment:
Amazing, I was thinking of writing something similar a while ago but never came round to actually doing it. But since there are also engine tests, I think just the path to the pester tests file would be better

Reply:
I have some ideas for this, including an argument completer. Let's flesh this out further in a separate PR. I'll revert this for this PR, and submit another one for improving the tools.